diff --git a/dist/vendor-node.caa511fc.js b/dist/vendor-node.caa511fc.js
index e11a4b8137f9486f878fb6c3da4b0067edd6a3e9..487775f1532e01445d6ea35a851457e482c76c0a 100644
--- a/dist/vendor-node.caa511fc.js
+++ b/dist/vendor-node.caa511fc.js
@@ -10511,6 +10511,7 @@ function getWorkerMemoryLimit(config) {
   return 1 / (config.maxThreads ?? getDefaultThreadsCount(config));
 }
 function stringToBytes(input, percentageReference) {
+  percentageReference = percentageReference || 100;
   if (input === null || input === void 0)
     return input;
   if (typeof input === "string") {